docs/features/software-catalog/kubernetes-in-backstage.md

```diff
@@ -78,7 +74,7 @@ cluster. Valid values are:
 The service account token to be used when using the `serviceAccount` auth
 provider.
 
-## RBAC
+## Roll Based Access Control
```

Comment:
```suggestion
## Role Based Access Control
```

Reply:
Clearly, I was on a roll 🎡 and missed this... 🙄 🤣